.github/workflows/release-notice.yml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) Contributors to the OpenEXR Project.
+
+name: Publish Release Notice to ASWF Slack
+
+on:
+  release:
+    types:
+      - published
+      # published should cover both 'released' and 'prereleased'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+    - name: 'Notify Slack #release-announcements'
+      id: slack1
+      with:
+        project_name: "OpenEXR"
+        slack_bot_token: ${{ secrets.SLACK_BOT_TOKEN }}
+        slack_channel: "#release-announcements"
+        project_logo: "https://artwork.aswf.io/other/aswf/text/aqua/aswf-text-aqua.png"
```

Comment:
You could change this to the logo for OpenEXR instead. It's best if it can be a square image, which the Icon only is pretty close to...

https://artwork.aswf.io/projects/openexr/icon/color/openexr-icon-color.png